core/src/main/java/org/elasticsearch/action/bulk/BulkProcessor.java

```diff
@@ -334,6 +325,10 @@ private boolean isOverTheLimit() {
         return false;
     }
 
+    private Runnable cancelTask(ThreadPool.Cancellable cancellable) {
+        return cancellable::cancel;
```

Comment:
can we fold this method into its only caller above? Also did we change the way we cancel the flush scheduler on close? Weren't we using FutureUtils#cancel before?